src/gcp_scanner/crawl.py

```diff
@@ -961,12 +922,9 @@ def get_service_accounts(project_name: str,
     request = service.projects().serviceAccounts().list(name=name)
     while request is not None:
       response = request.execute()
-
-      for service_account in response.get("accounts", []):
-        service_accounts.append(
-            (service_account["email"], service_account["description"]
-             if "description" in service_account else ""))
-
+      service_accounts=[(service_account["email"],service_account["description"]
+        if "description" in service_account else "")
```

Comment:
replace this `if ... else ...` with `.get("description", "")`

Reply:
replaced it with .get(key,"")